### PR TITLE
WEBDEV-5690 Adjust grid tile layout

### PR DESCRIPTION
**Pull request by latonv**
Removes some of the whitespace between lines on grid tiles, and ensures the stats row at the bottom is aligned consistently.

### CODE REVIEW

File: src/tiles/grid/tile-stats.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class TileStats extends LitElement {
             <p class="sr-only">Mediatype:</p>
             <mediatype-icon .mediatype=${this.mediatype}></mediatype-icon>
           </li>
-          <li class="col" title="${uploadsOrViewsTitle}">
+          <li class="col views" title="${uploadsOrViewsTitle}">
```
Comment (dualcnhq) [inline src/tiles/grid/tile-stats.ts:46]: views and favorites class is missing here, only reviews is declared in the CSS styles.
can we come up with a more generic class name for this instead?

File: src/tiles/grid/styles/tile-grid-shared-styles.ts [hunk 2]
```diff
@@ -35,14 +34,14 @@ export const baseTileStyles = css`
   }
 
   .item-info {
+    display: flex;
+    flex-direction: column;
```
Comment (dualcnhq) [inline src/tiles/grid/styles/tile-grid-shared-styles.ts:38]: :nice:

File: src/tiles/grid/tile-stats.ts (hunk 1 above)
Comment (latonv) [reply src/tiles/grid/tile-stats.ts:46]: We can certainly remove the unused "views" and "favorites" classes (unless we need to make different adjustments to those icons too).

For the reviews icon though, I am _specifically_ targeting that one for a tailored adjustment that likely doesn't generalize elsewhere...

File: src/tiles/grid/styles/tile-grid-shared-styles.ts (hunk 2 above)
Comment (latonv) [reply src/tiles/grid/styles/tile-grid-shared-styles.ts:38]: Flexbox is great 😄

**Pull request merged**